tests/test_types_leb128.py

```diff
@@ -0,0 +1,96 @@
+from dissect.cstruct.cstruct import cstruct
+
+
+def test_leb128_unsigned_read(cs: cstruct):
+    assert cs.uleb128(b"\x02") == 2
+    assert cs.uleb128(b"\x8b%") == 4747
+    assert cs.uleb128(b"\xc9\x8f\xb0\x06") == 13371337
+    assert cs.uleb128(b"~") == 126
+    assert cs.uleb128(b"\xf5Z") == 11637
+    assert cs.uleb128(b"\xde\xd6\xcf|") == 261352286
+
+
+def test_leb128_signed_read(cs: cstruct):
+    assert cs.ileb128(b"\x02") == 2
+    assert cs.ileb128(b"\x8b%") == 4747
+    assert cs.ileb128(b"\xc9\x8f\xb0\x06") == 13371337
+    assert cs.ileb128(b"~") == -2
+    assert cs.ileb128(b"\xf5Z") == -4747
+    assert cs.ileb128(b"\xde\xd6\xcf|") == -7083170
+
+
+def test_leb128_struct_unsigned(cs: cstruct, compiled: bool):
```

Comment:
```suggestion
def test_leb128_struct_unsigned(cs: cstruct):
```
We won't be testing if the structure gets compiled, because it won't be supported because of the unsupported type. So you can remove all references of `compiled`.